spec/controllers/moab_storage_controller_spec.rb

```diff
@@ -2,44 +2,6 @@
 
 RSpec.describe MoabStorageController, type: :controller do
 
-  describe "GET #index" do
-    it "returns http success status code" do
-      get :index
-      expect(response).to have_http_status(:success)
-    end
-    context 'assigns @stored_druids correctly' do
-      it 'without current or parent dot directories' do
-        get :index
-        expect(assigns(:stored_druids)).not_to include('.', '..')
-      end
-      it 'with all the fixture druids' do
-        get :index
-        expect(assigns(:stored_druids)).to include('bj102hs9687', 'bp628nk4868', 'bz514sm9647', 'dc048cw1328')
-      end
-      it 'Array of Strings' do
-        get :index
-        expect(assigns(:stored_druids)).to be_an_instance_of Array
-        expect(assigns(:stored_druids).first).to be_an_instance_of String
-      end
-    end
-    it 'assigns @storage_root correctly' do
-      get :index
-      expect(assigns(:storage_root)).to eq "#{Moab::Config.storage_roots}/#{Moab::Config.storage_trunk}"
-    end
-    it 'returns json by default' do
-      get :index
-      expect(response.content_type).to eq "application/json"
-    end
-    it 'returns json when requested' do
-      get :index, format: :json
-      expect(response.content_type).to eq "application/json"
-    end
-    it 'returns xml when requested' do
-      get :index, format: :xml
-      expect(response.content_type).to eq "application/xml"
-    end
-  end
-
```

Comment:
See above per making this a separate PR, and possibly just commenting out the route for now.